mappers/mqtt-mapper/go.mod

```diff
@@ -0,0 +1,41 @@
+module github.com/kubeedge/kubeedge-v1.17.0
```

Comment:
The name of the package is incorrect. You should set the mapper name to `mqtt` instead of `kubeedge-v1.17.0` when executing the `make generate` command to generate the mapper project.

Reply:
Thanks for your advice. I will remake a new mapper project using `make generate`.